docs/kb/api-version-3/prospects.md

```diff
@@ -245,6 +248,65 @@ XML responses to `create` requests are identical to `update` and `read` requests
 
 <a name="14833-updating-field-values" id="updating-field-values"></a>
 
+## [](#enpoints-batch-processing-)Endpoints for Batch Processing
+
+There are 3 endpoints available for batch processing up to 50 prospects at a time:
+
+/api/prospect/version/3/do/batchCreate
+
+/api/prospect/version/3/do/batchUpdate
+
+/api/prospect/version/3/do/batchUpsert
+
+These endpoints expect a POST or GET variable called "prospects" which holds either JSON or XML encoded data.
```

Comment:
For the most part I stole wording from something Ian forwarded me from Etienne:

"Endpoints available for batch processing:

/api/prospect/version/3/do/batchCreate
/api/prospect/version/3/do/batchUpdate
/api/prospect/version/3/do/batchUpsert

These endpoints expect a POST (or GET) variable called "prospects" which holds either JSON or XML encoded data in the following format:

[JSON and XML examples here]"